R/infrastructure.R

```diff
@@ -87,12 +87,12 @@ use_data <- usethis::use_data
 # `use_data` uses non-tidy NSE, so we cannot use it inside a function
 # hygienically. We could fix this with tidyevalation / rlang, but it seems
 # overkill just to get a deprecation message.
-#function(..., pkg = ".", internal = FALSE, overwrite = FALSE,
-                     #compress = "bzip2") {
-  #.Deprecated("usethis::use_data()", package = "devtools")
-  #usethis::use_data(..., internal = internal, overwrite = overwrite,
-                    #compress = compress)
-#}
+# function(..., pkg = ".", internal = FALSE, overwrite = FALSE,
```

Comment:
styler removed all function indentation as extra whitespaces, so I added it back. It will get removed each time styler is run against this file, though.